pyspecdata/core.py

```diff
@@ -3357,7 +3365,10 @@ def mean(self, *args, **kwargs):
                 "return_error kwarg no longer used -- use std kwarg if you"
                 " want to set the error to the std"
             )
-        return_error = process_kwargs([("std", False)], kwargs)
+        return_error, return_stderr = process_kwargs(
+            [("std", False), ("stderr", False)], kwargs
+        )
+        return_error |= return_stderr
```

Comment:
not has option to return std error vs std dev so when we can account for variability across multiple samples (e.g. our V measurements with the scope and AFG)

Reply:
If I'm understanding correctly, which is a bit off base -- I think that you want to refresh:

- what does `process_kwargs` do?
- what does the `|=` operator do?

and use that to explain what's up here

Reply:
Using process_kwargs it sees if you passed std or stderr as True. Then based on those it sets return_error or return_stderr to True (whaterver your kwarg was)

Reply:
Then the |= is an "in place OR operator" so it reassigns return_error to whichever was determined as True in the previous process_kwargs

Reply:
to be specific process_kwargs should be returning the tuple with those set to True or False